packages/angular_devkit/build_angular/test/browser/svg_spec_large.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { runTargetSpec } from '@angular-devkit/architect/testing';
+import { join, normalize, virtualFs } from '@angular-devkit/core';
+import { tap } from 'rxjs/operators';
+import { browserTargetSpec, host, outputPath } from '../utils';
+
+describe('Browser Builder allow svg', () => {
+
+  beforeEach(done => host.initialize().toPromise().then(done, done.fail));
+  afterEach(done => host.restore().toPromise().then(done, done.fail));
+
+  it('can use .component.svg as templates (uses raw-loader and does not copy .svg to dist)',
+    (done) => {
+
+    host.writeMultipleFiles({
+      './src/app.component.svg': `<svg></svg>`,
+      './src/main.ts': `declare var require: Function;
+      const x = require('./app.component.svg');
```

Comment:
This syntax is non-standard and not officially supported and shouldn't be part of a test.  This applies to the second test below as well.  The feature being added is SVG support within a Component template;  that use case is what should be tested.